packages/sign-client/test/sdk/persistance.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { generateRandomBytes32 } from "@walletconnect/utils";
```

Comment:
Sorry to be that guy, but better we fix it up front: Should be `persistence.spec.ts` (all `e`) 😬